hardware-api.md

```diff
@@ -150,8 +150,125 @@ uart.pipe(process.stdout);
 ```
 
 ## Button and LEDs
+There are 4 LEDs available on the Tessel 2, you may see them labeled on the board as `ERR`, `WLAN`, `LED0`, and `LED1`. They are available through the `tessel.led` object. 
 
-Tessel 2's button and LEDs are not yet exposed in the API – but you can change that! See [#15](https://github.com/tessel/t2-firmware/issues/15) for a description of what needs to be done.
+```js
+// an array of available LEDs
+var leds = tessel.led;
+
+// ERR - Red
+var red = leds[0];
+
+// WLAN - Amber
+var amber = leds[1];
+
+// LED0 - Green
+var green = leds[2];
+
+// LED1 - Blue
+var blue = leds[3];
+```
+
+Each LED has a few methods for controlling it.
+
+
+```js
+// Green LED
+var led = tessel.led[2];
+
+/*
+* Method: write
+* Arguments:
+* - value: usually 0 or 1 but could be any value representing true or false
+* - callback: function to call once the led's value has been set and is passed an error object if one occured.
+*
+* Sets the value of the led. Truthy values (1, true, 'string') turn it on. Falsey values (0, false) turn it off.
+*
+* Aliases:
+* - output
+*/
+led.write('1', function (err) {
+  if (err) {
+    console.log('There was an error with turning on the led.', err);
+  } else {
+    console.log('The green led is now on!');
+  }
+});
+
+/*
+* Method: read
+* Arguments:
+* - callback: function to call once the led's value has been read and is passed the value of the led.
+*
+* Reads the set value of the led.
+*/
+led.read(function (value) {
+  console.log('The value of the green led, probably 1:', value);
+});
```

Comment:
I'd rather not advertise the existence of this method. LED state is normalized upon initialization and stored internally now, so the `value` property returns the internal state—which means there is no reason for user code to _want_ to use an API like this. Take a look: 

- https://github.com/tessel/t2-firmware/blob/master/node/tessel-export.js#L932-L937
- https://github.com/tessel/t2-firmware/blob/master/node/tessel-export.js#L885-L893 <-- preferred

I don't think this should be included in the published documentation, because it's not useful. It's existence here is solely for backward compatibility with T1 code